### PR TITLE
Added new pages and stac logo.

### PR DESCRIPTION
**Pull request by tusharrao198**
Issues  . #142   , #148  
1.Created pages
    1. about.html, 2. astrax.html , 3. zenith.html , 4. utkarsh.html, 5. alumni_s.html, 6. photogallery.html 7. videogallery.html 
    added urls in urls.py 

2. Modified navbar.html, created a dropdown list of events of Gallery.
 
3. added stac logo and adjusted size in style.css 

4. Title shown for active page. For eg. "STAC | IIT Mandi - 'page_name' " Now it will be visible in title on which page we are active. 

5. changed {% extends 'base.html' %}   to  {% extends 'stac_iitmandi/base.html' %}  to avoid confusion from which app we are importing.

6. Added some bootstrap links in base.html for navbar dropdown purpose (like show dropdown on hovering. )

### CODE REVIEW

File: src/stac_iitmandi/templates/stac_iitmandi/base.html [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
+  <style type="text/css">
+  /* ============ desktop view ============ */
+  @media all and (min-width: 992px) {
+      .navbar .nav-item .dropdown-menu{ display: none; }
+      .navbar .nav-item:hover .nav-link{ color: #010101;  }
+      .navbar .nav-item:hover .dropdown-menu{ display: block; }
+      .navbar .nav-item .dropdown-menu{ margin-top:0; }
+  }	
+  /* ============ desktop view .end// ============ */
+  </style>
```
Comment (LalitNM) [inline src/stac_iitmandi/templates/stac_iitmandi/base.html:41]: Don't you think this should be in `style.css`?

File: src/stac_iitmandi/templates/stac_iitmandi/base.html [hunk 2]
```diff
@@ -1,17 +1,45 @@
...
   <!-- Custom CSS -->
   <link rel="stylesheet" type="text/css" href="{% static 'stac_iitmandi/style.css' %}">
+  
+  <!-- jQuery  for dropdown hover in Navbar -->
+  <script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44="
+  crossorigin="anonymous"></script>
+
+  <!-- Bootstrap files (jQuery first, then Popper.js, then Bootstrap JS) for dropdown hover in Navbar-->
+  <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.bundle.min.js" type="text/javascript"></script>
```
Comment (LalitNM) [inline src/stac_iitmandi/templates/stac_iitmandi/base.html:30]: JS cdns should be at the bottom of the page(we already have  those).

**Review by LalitNM — CHANGES_REQUESTED** [review]
In addition to the changes requested check the functioning of drop down on small-screened devices. It is not working.

File: src/stac_iitmandi/templates/stac_iitmandi/base.html (hunk 1 above)
Comment (tusharrao198) [reply src/stac_iitmandi/templates/stac_iitmandi/base.html:41]: Yes , I forgot to add it. I will add it in style.css

File: src/stac_iitmandi/templates/stac_iitmandi/base.html (hunk 2 above)
Comment (tusharrao198) [reply src/stac_iitmandi/templates/stac_iitmandi/base.html]: updated

File: src/stac_iitmandi/templates/stac_iitmandi/base.html (hunk 1 above)
Comment (tusharrao198) [reply src/stac_iitmandi/templates/stac_iitmandi/base.html]: Drop down error fixed.
and css moved to style.css

File: src/stac_iitmandi/views.py [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
+    return render(request, 'stac_iitmandi/astrax.html', {'title': 'Astrax'})
+# rendering Utkarsh page
+def utkarsh(request):
+    return render(request, 'stac_iitmandi/utkarsh.html', {'title': 'Utkarsh'})
+# rendering Zenith page
+def zenith(request):
+    return render(request, 'stac_iitmandi/zenith.html', {'title': 'Zenith'})
+# rendering Alumni's page    
+def alumni_s(request):
+    return render(request, "stac_iitmandi/alumni_s.html", {'title': "Alumni's"})
```
Comment (gajrajgchouhan) [inline src/stac_iitmandi/views.py:21]: I think `alumni_s` could be changed to `alumni`, and maybe the `Alumni's` to `Alumni` in navbar and here too.

File: src/stac_iitmandi/templates/stac_iitmandi/base.html [hunk 4]
```diff
@@ -1,17 +1,45 @@
...
 <html lang="en">
 
 <head>
-  {% load static %}
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>{% block title %}STAC | IIT Mandi{% endblock %}</title>
+  <!-- Title of Page -->
+  {% if title %}
+    <title > STAC | IIT Mandi - {{title}}</title>
```
Comment (gajrajgchouhan) [inline src/stac_iitmandi/templates/stac_iitmandi/base.html:11]: I think it should be only `{{title}}` for better view.

**Review by gajrajgchouhan — CHANGES_REQUESTED** [review]

File: src/stac_iitmandi/templates/stac_iitmandi/_navbar.html [hunk 5]
```diff
@@ -1,27 +1,40 @@
...
-        <li class="nav-item">
-          <a class="nav-link" href="#">ZENITH</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="#">BLOG</a>
-        </li>
-        <li class="nav-item">
-            <a class="nav-link" href="#">ABOUT</a>
-        </li>
+    <div class="collapse navbar-collapse" id="main_nav">
```
Comment (gajrajgchouhan) [inline src/stac_iitmandi/templates/stac_iitmandi/_navbar.html:8]: I think the `id` is not supposed to be changed from `navbarSupportedContent`.

File: src/stac_iitmandi/views.py (hunk 3 above)
Comment (tusharrao198) [reply src/stac_iitmandi/views.py:21]: Ok will do that.

**Review by gajrajgchouhan — CHANGES_REQUESTED** [review]

File: src/stac_iitmandi/templates/stac_iitmandi/base.html (hunk 4 above)
Comment (tusharrao198) [reply src/stac_iitmandi/templates/stac_iitmandi/base.html:11]: I think site name should be necessary with title.
@LalitNM  , What is your opinion on this?

File: src/stac_iitmandi/templates/stac_iitmandi/_navbar.html (hunk 5 above)
Comment (tusharrao198) [reply src/stac_iitmandi/templates/stac_iitmandi/_navbar.html:8]: But it is of no use until now.

Comment (LalitNM) [reply src/stac_iitmandi/templates/stac_iitmandi/_navbar.html:8]: > But it is of no use until now.

Then you should leave it as it is.

File: src/stac_iitmandi/templates/stac_iitmandi/base.html (hunk 4 above)
Comment (LalitNM) [reply src/stac_iitmandi/templates/stac_iitmandi/base.html:11]: It should be like `<title of page> | STAC | IIT Mandi`, e.g. `Zenith | STAC | IIT Mandi`.

**Review by LalitNM — APPROVED** [review]

**Comment by LalitNM** [comment]
@tusharrao198, one good practice is to create new branch for each pull request. You can delete this branch later.

**Comment by tusharrao198** [comment]
All changes requested are made in the latest commit.

**Comment by LalitNM** [comment]
@tusharrao198, for me this looks great :clap:, let's see what @gajrajgchouhan says.

File: src/stac_iitmandi/templates/stac_iitmandi/_navbar.html [hunk 6]
```diff
@@ -1,27 +1,40 @@
-
+{% load static %}
 <nav class="navbar navbar-expand-lg sticky-top bg-custom navbar-light mt-5">
-    <a class="navbar-brand" href="#">STAC</a>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+<!--<nav class="navbar navbar-expand-lg navbar-dark bg-primary">-->
```
Comment (gajrajgchouhan) [inline src/stac_iitmandi/templates/stac_iitmandi/_navbar.html:3]: We don't need this comment.

**Review by gajrajgchouhan — DISMISSED** [review]

File: src/stac_iitmandi/templates/stac_iitmandi/base.html (hunk 4 above)
Comment (gajrajgchouhan) [reply src/stac_iitmandi/templates/stac_iitmandi/base.html]: We can probably do something like `{{title}} - STAC | IIT Mandi`.

File: src/stac_iitmandi/templates/stac_iitmandi/_navbar.html (hunk 6 above)
Comment (LalitNM) [reply src/stac_iitmandi/templates/stac_iitmandi/_navbar.html]: fixed in 96587951fe63db5be7385b1b9f60e6ba12312d21

**Comment by LalitNM** [comment]
This looks great :clap: @tusharrao198. merging this :rocket:

**Pull request merged**